core/health/api/pom.xml

```diff
@@ -11,4 +11,10 @@
     <artifactId>org.opennms.core.health.api</artifactId>
     <packaging>bundle</packaging>
     <name>OpenNMS :: Core :: Health :: API</name>
+  <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
Must this dependency reflected in the feature/bundle dependencies, too?

Reply:
no need, should be available by default in karaf i think

Reply:
When I remove that dependency the build fails.